source/content/partials/drush-compatibility.md

```diff
@@ -8,36 +8,30 @@ tags: [--]
 reviewed: ""
 ---
 
-Drush only supports one install method. Your Drupal site is required to be built with Composer and your Drush must be compatible and listed as a dependency. 
-
-See the [Drush Drupal Compatibility chart](https://www.drush.org/latest/install/#drupal-compatibility) for version compatibility information. Consider the following information when determining which Drush version is best suited for your site:
-
-* Drush 10 is available with the [addition of the  `pantheon.yml` file](#configure-drush-version), or for [site-local installation](#site-local-drush-usage). It requires Drupal 9 or higher, [Composer](/guides/composer/), and PHP 7.1 or higher.
+<Alert title="Note" type="info">
 
-* [Drush 9](https://docs.drush.org/en/9.x/) and Drush 10 only work on Drupal 8.4 and higher.
+We highly recommend running Drush version 8.3.2 or higher when running Drush locally.
```

Comment:
I suspect this is true for Drupal 7, but we should include a note to say something like "for modern versions of drupal we recommend running the most recent version of Drush that is compatible with your site."